src/go/rpk/pkg/tools.go

```diff
@@ -0,0 +1,8 @@
+// +build tools
```

Comment:
Looks good. Can you add a comment explaining the reason for this file. 

For example, why couldn't we do go get crlfmt at specific hash

Reply:
The whole point behind `go.mod` is so we don't have to do `go get` ourselves.

This file (as explained in the specific commit msg) is a placeholder for dependencies that are not needed by the actual code.
Since they do not appear in any files, operations like `go mod tidy` removed the dependency from go.mod altogether.
The reason we need to have it in the go.mod is to control which version clients install ( so we don't end up updating and messing with our formatting conventions -which is what happened previous week).